### DISCUSSION TITLE
Variable Submenu should be always accessible on the screen like Navigation

### DISCUSSION THREAD

**Discussion by PengWang-SMARTM** [grafana/grafana#56057] [Dashboards, Panels & Visualizations]
I created a dashboard with a number of panels that cannot fit into one screen page.

When I go to the bottom of the page, the variable submenu is scrolled up along with the panels. Only the dashboard navigation header (time picker) is still available on the top of the screen.

If I need to change the variables, I have to scroll up to get to the submenu and then scroll down again to see the changes in the panel. 

If the variable submenu can stay on the top, it would be much more convenience.

To be one more step further, it would be even better if the submenu and navigator can support autohide.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
